src/js/components/LanguageWrapper.js

```diff
@@ -78,7 +78,6 @@ class LanguageWrapper extends Component {
           <div>
             <Route path="/" render={props => (
               <section>
-                <Banner />
```

Comment:
Can we delete the banner module from the repo if we're not gonna use it? Or maybe this is a temp thing.

Reply:
👍